### ISSUE TITLE
A higher result

### ISSUE THREAD

**Issue by weixmath** [jongchyisu/mvcnn_pytorch#3]
Hello ！I have a new problem that the resnet50 architecture get 96.1% ish accuracy twice! I set the parameters as what you mentioned in your paper. I keep all the default settings and only change the num_models to 0 and learning_rate in second stage to 1e-5. Did you get such high results in your runs?

**Comment by jongchyisu** [comment]
That's nice! I didn't really fine-tune those hyper-parameters like the learning rate decay, and I only got 95.5% before.

**Comment by weixmath** [comment]
ok thank u  👍

**Comment by dimrecon** [comment]
hello 
could u pls tell how many epochs to run for my train input ( 82665 models with 20 views) as pretrained true ? And how much is the expected time for completion of running training?

**Comment by jongchyisu** [comment]
30 epochs for each stage, and takes about 1 day in total.

**Comment by dimrecon** [comment]
for me one epoch takes more than 14 hrs. How can use your model effectively. what changes are to be applied in your codes for reducing running time? Pls help.

**Comment by dimrecon** [comment]
> 30 epochs for each stage, and takes about 1 day in total.

**Comment by jongchyisu** [comment]
I'm using the ModelNet40.
